### DISCUSSION TITLE
My private repository is visible for everybody

### DISCUSSION THREAD

**Discussion by spuelm** [community/community#131558] [New to GitHub]
Hi, 
I created a repository with "visibility private" but it seems to be accessible without even beeing logged in to gitHub.
https://github.com/spuelm/pdfshuffle.git
Any ideas what to dso to make it real private so nobody can read my files?
Thanks for your help.

**Comment by mustafaobaidd** [comment]
1. **Check Repository Settings:**
   - Go to your repository on GitHub.
   - Click on the "Settings" tab.
   - Ensure that under "Options," the repository visibility is set to "Private."

2. **Verify Access:**
   - Log out of your GitHub account or use an incognito browser window.
   - Try to access the repository URL to confirm it is not accessible.

3. **Review Collaborators:**
   - In the "Settings" tab, go to "Manage access" under the "Access" section.
   - Check the list of collaborators and teams with access to the repository. Ensure that only the intended people have access.

4. **Token and Link Sharing:**
   - Make sure you haven’t shared any access tokens or links that grant access to the repository.
   - If you suspect an access token might have been compromised, revoke and regenerate it.

5. **GitHub Support:**
   - If everything seems correct but the repository is still accessible publicly, contact GitHub Support for assistance. There might be a bug or an issue with your account settings.
   - 
![Screenshot 2024-07-09 194624](https://github.com/community/community/assets/145908857/5e025f1d-9e55-4079-a2df-6cd9d74df07f)

**Comment by spuelm** [comment]
Thanks it works

**Reply by mustafaobaidd** [reply]
You are welcome. Please make my answer, the answer to the question so that everyone can benefit
